internal/driver/driver.go

```diff
@@ -37,6 +37,8 @@ type Conn struct {
 	r *bufio.Reader
 	w *bufio.Writer
 	l *slog.Logger
+
+	lastRequestID int32
```

Comment:
I think it should be a package-level `atomic.Int32` to easily differentiate between the requests of different clients.

Reply:
I didn't use atomic, as the Conn is not safe for concurrent use, but when I move it to package level, yes sure